### ISSUE TITLE
Weekly Community Check-in #19 - Express gratitude :-)

### ISSUE THREAD

**Issue by sagarpreet-chadha** [publiclab/leaflet-blurred-location#198]
Hello Everyone!!!

We all at Public Lab :balloon:  - learn, grow, work, brainstorm ideas, contribute together so why not share about our weekly goals and the awesome work we have done at Public Lab with each other, so we can support and collaborate with each other better. The goal of our Community Check-In is for every community member to share something about their work from the past week and about their current week's goal dart. You are also welcome to share fun-fact :smile: , new ideas :bulb: , your learning goals :ballot_box_with_check:

If you're new here, welcome :balloon: and please comment a Hello message below, we love to work with you. If you're looking for new issues, please try some of our first-timers-only issues.
You can find them at https://code.publiclab.org/#r=all

We're SO EXCITED to have your help!

Is there anything, you would like to share with us from last week's work? What is your plan for this week?

If you have not planned yet, just leave a Hello! so that we know that you are in sync with us :arrows_clockwise: and doing well!

As always, if you're waiting for a review, or if you're stuck, you can request help here, leave a comment on your issue with @publiclab/reviewers mentioned, or ask the community in our Gitter chatroom :open_hands:

**Gitter**

Gitter is an active chatroom in our community and we'll be sending weekly reminders about check-ins there. Be sure to sign up there for these updates or just to join the conversation page.

## **This Week's Theme :** 

**Express gratitude** : This week let us remember our first contribution to the open-source world and how we felt after we got our first Pull Request merged 😍 .
Let us thank PublicLab community for welcoming us and giving us this awesome experience of contributing along with so many experienced developers from all over the world 🌏 .
- You can share your first open-source experience .
- You can thank the people who mentored you .
- Or just say Hi :-) 

You can find our list of previous check-ins [here](https://github.com/publiclab/plots2/issues/5627) :notebook: 

### Google Season of Docs: 
 
**Applications for mentor (Google Season of Docs) are now open --- check out this exciting new program here :** https://github.com/publiclab/plots2/issues/5550

**Community Survey**

@ebarry has compiled the results of the community survey, which was open from January 14 to March 21, into a comprehensive code community report. Please add your comments here to this thread for what we are inspired to do/change as a result of reading:[https://publiclab.org/notes/liz/04-15-2019/report-2019-software-contributors-survey]( https://publiclab.org/notes/liz/04-15-2019/report-2019-software-contributors-survey)

This is a really unique opportunity to contribute to open source from a greater scope, and a reminder of the greater meaning behind the work all of you have been doing!! 

cc
@mohitRJranjan @milaaraujo @kevinzluo @Rishabh570 @sashadev-sky @geekychaser @shubhscoder @sidntrivedi012 @SidharthBansal @cesswairimu @stefannibrasil @ViditChitkara @tech4GT @mridulnagpal @IshaGupta18 @CoderJolly @namangupta01 @sagarpreet-chadha @Souravirus @MayankKashyap @siaw23 @ryzokuken @icarito @steviepubliclab, @ebarry, @jywarren, @sagarpreet-chadha, @JonathanXu1, @uzorjchibuzor, @eli6, @rexagod, @divyabaid16, @dinaelhanan, @oorjitchowdhary, @wanzulfikri, @mohitRJranjan @publiclab/mentors @publiclab/image-sequencer-guides @publiclab/leaflet-environmental-layers-guides @Paarmita @publiclab/reviewers @Divy123 @bhavayAnand9 @Mridul97 @avsingh999 @romanrodriguez @Dhiraj240 @codeIriss @rishabhc32 @chirag-singhal @IgorWilbert @dhruvgupta167 @vrk99 @pdurbin @HarshKhandeparkar @aashna27 @Harshithpabbati @marieram @swiatek7 
@niravasher

Thanks and keep contributing! Contributing to Public Lab means contributing to the Environment.

People who did the check-ins-

1. Gaurav Sachdeva (@gauravano) https://github.com/publiclab/plots2/issues/4327
2. Sidharth Bansal (@SidharthBansal) https://github.com/publiclab/plots2/issues/4369
3. Cess (@cesswairimu) https://github.com/publiclab/plots2/issues/4476
4. Harman Jolly (@CoderJolly) https://github.com/publiclab/plots2/issues/4548
5. Rishabh Rawat (@Rishabh570) https://github.com/publiclab/plots2/issues/4607
6. Sasha Boginsky (@sashadev-sky) https://github.com/publiclab/plots2/issues/4676
7. Rishabh Kr Bothra (@geekychaser) https://github.com/publiclab/plots2/issues/4711
8. Shubham Sangamnerkar https://github.com/publiclab/plots2/issues/4757
9. Siddhant N Trivedi (@sidntrivedi012) https://github.com/publiclab/plots2/issues/4784
10. Isha Gupta (@IshaGupta18) https://github.com/publiclab/plots2/issues/4844
11. Gaurav Sachdeva (@gauravano) https://github.com/publiclab/plots2/issues/4879
12. Harshith Pabbati (@Harshithpabbati) https://github.com/publiclab/plots2/issues/4917
13. Gaurav Sachdeva (@gauravano) https://github.com/publiclab/plots2/issues/5034
14. Harshith Pabbati (@Harshithpabbati) https://github.com/publiclab/image-sequencer/issues/883
15. Harman Jolly (@CoderJolly) https://github.com/publiclab/mapknitter/issues/447
16. Divya Baid (@divyabaid16) https://github.com/publiclab/PublicLab.Editor/issues/317
17. Gautami Gupta (@gautamig54) https://github.com/publiclab/leaflet-environmental-layers/issues/164
18. Sasha Boginsky (@sashadev-sky) https://github.com/publiclab/Leaflet.DistortableImage/issues/215
19. Sagarpreet Chadha (@sagarpreet-chadha) https://github.com/publiclab/leaflet-blurred-location/issues/198

Have a great week everyone :balloon: :tada: :sparkler:

**Comment by HarshKhandeparkar** [comment]
@sagarpreet-chadha Emojis and other markdown doesn't work in titles.

**Comment by sagarpreet-chadha** [comment]
@marieram @swiatek7 @gauravano @publiclab/reviewers @publiclab/is-reviewers @publiclab/community-reps

@subhahu123 @An0u @Jaya738 @mgroovyank @shreyateeza @TANGL3SIT3S @nazeem1988 @karunagoyalk @Nirvikalpa108 @cassianoblonski @EngrEric @Anjalizi @GettyOrawo @coderbeetle @corbettbw @govindjeevan @gautamig54 @daddyjab @emestabillo @letnewKid @srujana121 @jazdao @ChinwenduSilvia @aggarwal19 @Jenniline @madelinejones @helenatxu @richa031 @fabsar @kuja24 @megha070 @Aarabhi2017 @Manvi07 @siddhishree @Julius26 @VinneyJ @sonali9696 @rarrunategu1 @coreytegeler @labseven @milomacphail @CleverFool77 @ananyaarun @ananya @divyabaid16 @publiclab-mimi

**Comment by sagarpreet-chadha** [comment]
@shubhi2000 @Priyak5 @vaarigupta @techchic @ananya @hc-barker @edwd42 @PritiShaw @themonster2015 @supriya-kotturu @AnthoniaOkafor @jtmims @scheleon @mahmodHammad @santushk @jillpena @stoic-plus
@vaarigupta @alonpeer @madeofhuman

**Comment by sagarpreet-chadha** [comment]
@ibnjunaid @HarshKhandeparkar @aashna27 @Harshithpabbati @niravasher @mmmelissa

**Comment by sagarpreet-chadha** [comment]
This week's theme is inspired from open call 😄 !
I will start , I would like to especially thanks @jywarren along with @gauravano and @SidharthBansal for helping me out whenever i get stuck . 
Thank you for making my experience with PublicLab so awesome 🙌 🎈 💃  .

⭐️ ⭐️ ⭐️

**Comment by cesswairimu** [comment]
Many thanks to @jywarren and @gauravano :1st_place_medal:  :1st_place_medal:  for guiding me through my first issue.
I had contributed to other open source projects but found this experience super friendly(and still is) and everyone is so polite and willing to help :raised_hands: :raised_hands: 

Awesome theme @sagarpreet-chadha :tada: :tada: 
Great week ahead and nice to see you all today :smile:

**Comment by HarshKhandeparkar** [comment]
I would like to open the next check-in
cc @gauravano @jywarren @sagarpreet-chadha

**Comment by sagarpreet-chadha** [comment]
Yes awesome @HarshKhandeparkar  👍 👍 👍

**Comment by jainaman224** [comment]
I would like to thank @gauravano, @ViditChitkara, @namangupta01 and @sagarpreet-chadha for inspiring me to be part of such an amazing community.

**Comment by AnthoniaOkafor** [comment]
I want to thank @jywarren and @gauravano for their effort at making this community remarkable. My first PR was tough but @gauravano saw me through it.

**Comment by sashadev-sky** [comment]
Aw love this theme! I'd like to express gratitude to PL for providing my first open source experience, and so much more along the way as I have grown in the community for 7 months now! 

@gauravano for always answering all my questions, @jywarren for always making time when I need to discuss a PR and teaching me new things, @ebarry for welcoming me to co-work at the PL NY office and being so warm ❤️ @icarito for keeping our CI running and fixing it literally moments after it breaks @rexagod for laboring with me on DistortableImage bugs 😆, @cesswairimu thank you for showing me browser stack! 

Additionally so many other people who have helped review my PRs in the past. Most recently, @divyabaid16 @shivam15

**Comment by gauravano** [comment]
Great theme @sagarpreet-chadha! 

Public Lab is the community with which I started my Open Source journey some 1 year 7 months ago. I still remember my first PR - https://github.com/publiclab/plots2/pull/1642 where I did all the silly mistakes :see_no_evil: , and @jywarren helped me out. 

I got support from various people during this journey @jywarren @sagarpreet-chadha @SidharthBansal  @Souravirus @namangupta01 @ViditChitkara @icarito @ebarry @steviepubliclab and many more.

I learned many things being part of this community - wrote my first test here only, API development, testing, and each PR which I reviewed, collaborated with someone was a learning experience.
 
Thanks to Public Lab for being a crucial part of my life, @jywarren for helping out and being an awesome mentor and to all the contributors who made this journey possible.

Thank you @sagarpreet-chadha @cesswairimu @jainaman224 @AnthoniaOkafor @sashadev-sky :heart:  

Have a great week ahead everyone :100:

**Comment by sashadev-sky** [comment]
These are so great, can we keep this one pinned forever 💯

**Comment by divyabaid16** [comment]
Hello everyone!!
It's a great theme @sagarpreet-chadha :smile: 
It has been around 1.5 months that I have been contributing in Public Lab and I'm having an amazing experience. Everyone is so friendly and helpful here. It seems like I have been here for months and everything is going very smoothly.
I remember, the first issue for me was opened by @cesswairimu and she helped and guided me through the time my first PR got merged. I'm very thank to you as you were the first person I interacted here. :100: 
I would like to thank each and everyone who belongs here and have helped each other in every possible way guiding, reviewing, helping and also motivating. 
In my initial days, when I was a newbie bee here, I was suppoerted and encouraged by my mentors and a special and heartful thanks to them :balloon: :tada: :smile: 

This week, my end semester exams are beginning, so I won't be able to contribute much but will definately join by full force after 11 May.

Thanks!

**Comment by Divy123** [comment]
Greatings everyone!!
Awesome theme @sagarpreet-chadha . Thanks a lot for opening this.
I have been contributing to Public Lab for six months and I never felt a single day of my life when I did not get something from this beautiful community.

I would like to thank @jywarren for always helping me out despite being so busy, @tech4GT who always stood for us, the contributors and helped us to learn even more always, @HarshKhandeparkar an open source enthusiast in all dimensions who was always super active to help us solve the issues in an even better way, @gauravano who was always there to help me in case of any confusions, @aashna27 for always helping me by reviewing the PRs and helping solve the issues efficiently, @SidharthBansal for being very fast with his helpful suggestions and all other contributors and mentors for making Public Lab such a modest and hard working organisation.

**Comment by shivam15** [comment]
Hi All !
I have not contributed much but just two times. It was a great experience and would look forward to contribute more. 😃
I would like to thank @sashadev-sky  and @gauravano for helping in my first PR.

**Comment by gautamig54** [comment]
This is such a wonderful theme. There is so much to write here. 
@sagarpreet-chadha Thank you!
Public labs is my first ever open source experience and has made me fall in love with this domain. I remember how uncertain I was, if open source is my place, but by being here I realise this is exactly where I wanted to be. Everyone here, be it mentor or fellow contributors, have always been so humble and helpful.
Starting with @jywarren, @gauravano and @cesswairimu, who helped me through my first PR. Thank you so much for being such supportive and motivating mentors. Your guidance have been an important part in journey of learning. It was a joyous moment to see my first PR getting merged. 
Thank you @siaw23, @sashadev-sky, @namangupta01 for helping me with my issues and PRs.
Most importantly, @IshaGupta18, @CleverFool77, @divyabaid16, @ananyaarun Thank you all. Its been amazing collaborating with you guys. Be it issues or PRs, your suggestions have always been helpful. It was fun dividing up tasks and working together on them. Cheers girls! <3 
The humility and collaboration between contributors and mentors that exists in Public labs is its strength and makes me wanna contribute here as much as I can. 
Kudos to everyone here!!!!
Hope to have a great journey here.
Thank you everyone!!!

**Comment by jywarren** [comment]
Hello everyone, this post makes me feel so emotional! I am endlessly grateful to you all for making this such a wonderful place, for giving me such renewed faith in fellow people, and strength and inspiration. It always brings a smile to my face to see people helping and cheering one another in the many comments and issues we work on together. I'm always cheered to see a newcomer have a good experience and to see people then turn around and extend that to someone else in turn. 

I wanted this week to express extra gratitude to @Souravirus who has been tirelessly updating, rebasing, and refining one of the biggest PRs we've ever seen, a conversion of our whole site to Bootstrap 4: https://github.com/publiclab/plots2/pull/3937/

@Souravirus is always kind, thoughtful, and never complains that one more PR is merged causing another merge conflict to be required in this immense project. So, I am looking to do no more and to merge it tomorrow, and publish it to https://stable.publiclab.org.

Some things may break. Some will need adjustment. But this is something we can take on together and solve as a team. 

Thank you so much @Souravirus for your efforts here!!! And once again to everyone who's been pitching in across our projects. You're what makes this all possible! ❤️ 💞 🚀 🙌 🎉

**Comment by Souravirus** [comment]
Hi everyone,
Thanks, @jywarren for the special gratitude. It means a lot to me.

I really want to express gratitude to everyone from the starting of my journey with Public Lab till now. I started with a first-timers-issue issue and I remember how quick response came from @gauravano. I never expected this from such a big organization.

Then I want to thank @sagarpreet-chadha who really helped me by reviewing my PRs when I was in my early stage of contributing.

I also want to thank @jywarren, @siaw23 who helped me with the rails upgrades in my GSoC period. I really didn't know much about upgrades but the way @siaw23 helped me with the rails 5.0 upgrades that is really commendable.
At last thanks to everyone for becoming part of such a beautiful community.

**Comment by jywarren** [comment]
This was JUST MERGED!!!!! ❤️ 🙌🙌🙌🙌 ⚡️⚡️⚡️⚡️

**Comment by jywarren** [comment]
Congrats, @Souravirus !!!!

It'll appear shortly on https://stable.publiclab.org and we can take the next steps.

**Comment by sashadev-sky** [comment]
> Hi All !
> I have not contributed much but just two times. It was a great experience and would look forward to contribute more. 😃
> I would like to thank @sashadev-sky and @gauravano for helping in my first PR.

@shivam15 thank you for helping me with my own PR afterwards!!

**Comment by skilfullycurled** [comment]
Hi everyone, sorry to be late to the gratitude party, but I love this check-in!  I hope I've gathered everyone who helped me this past week even if it was just a note of welcome.  If I missed anyone, please forgive me and feel free to respond with a "don't forget me!".  First, I want to thank everyone in the largest sense.  Although this week was my first contribution to the site, I've participated here long enough to know that the more specific thank you's I'm about to give are made possible by a community which actively chooses to maintain specific values and culture.  So to everyone who makes this community possible, you have my gratitude.  To get more specific:

**Comment by skilfullycurled** [comment]
Hold on, I'm minimizing files...

**Comment by skilfullycurled** [comment]
Okay.  Enjoy!  [web_dev_2016_2019.pdf](https://github.com/publiclab/leaflet-blurred-location/files/3119036/web_dev_2016_2019.pdf)

**Comment by skilfullycurled** [comment]
PS: can someone ELI5 why the check-in is in a different place each time as opposed to just being on plots2?  Not a criticism, just peculiar.

**Comment by gauravano** [comment]
> Okay. Enjoy! web_dev_2016_2019.pdf

It's awesome :heart_eyes: !!

We open check-in in different repos every week so that each repo gets attention, increasing activity, contributors might end-up exploring that project :sweat_smile:. Yeah, it may become tough for a user to find the check-in issue.

**Comment by skilfullycurled** [comment]
I love that!  I opened an [issue](https://github.com/publiclab/plots2/issues/5624) with an idea about how we might balance the benefits of doing so while still having a central list for people to find the current and past ones.  I'm not sure the process for resolving proposals, but if people can take a look and weigh in at some point, that'd be great.  I now return us to our regularly scheduled gratitude.

**Comment by ebarry** [comment]
Wow this visualization of the range of subjects (tags) involved in + the
degree of interconnectivity of web development with the whole of Public Lab
is very expressive

Thanks Benjamin!

**Comment by skilfullycurled** [comment]
Thanks @ebarry!  Completely agreed, I think it really captures the push for outreach and growth in the community.  What I also find fascinating are the changes in tool focus and in one instance a convergence between two connected by data logging.  

To me, it really speaks to why the work on the [tags interface](https://publiclab.org/notes/edie_blues/01-09-2019/the-tags-page-mockup-let-me-know-what-you-think) that edie_blues (forgive me I can't find their github username) has been working on, and the newly implemented [tag graph](https://github.com/publiclab/plots2/issues/1502) will be really useful.

I hope the Q&A example will also help us understand engagement, and it's actually the visualization that made me think about interfacing this site and the Q&A on the publiclab.org because it really didn't represent all of the Q&A's that are happening over here!

More in a write up.

**Comment by IshaGupta18** [comment]
Thank you for this lovely theme @sagarpreet-chadha!

I would like to thank everyone who has worked with me and have helped me in
every step of this amazing journey. I couldn't have started my journey
without @jywarren, @SidharthBansal, @gauravano who got my first PR merged
and everyone who took me forward. @cess, has been ever-helpful and I am
sure I am missing out tons of names here, but I am truly grateful for all
the support I have received. And a very special thanks to @arpansac, who
has always helped and guided me whenever I got stuck! Thanks all!

**Comment by gauravano** [comment]
Hi @HarshKhandeparkar, you would be opening next check-in right?

**Comment by rexagod** [comment]
@gauravano I'd like to open the next check-in after @HarshKhandeparkar's if no one's already booked that spot. Thanks!

**Comment by skilfullycurled** [comment]
Also, @gauravano, would it be possible to add issue 5627 to the template so all new check-ins will reference the community check-ins list?  For background see the issue [here](https://github.com/publiclab/plots2/issues/5624).

Maybe after the explanation of this weeks check-in?  

> "You can find the list of links to previous weeks check-ins [here](issue 5627)."

Or perhaps with the "new people" welcome?  I put the additional text in bold just to highlight it now, I'm not saying the text should actually be in bold.  As an aside, note that the phrase "You can find them at" does not have a link.

> If you're new here, welcome 🎈 and please comment a Hello message below, we love to work with you. **You can find our list of previous check-ins [here](issue 5627).**  If you're looking for new issues, please try some of our first-timers-only issues.
> You can find them at **<--missing link**

Anyone else have any ideas where that might go?

**Comment by HarshKhandeparkar** [comment]
I will open this week's check-in in the evening.

**Comment by gauravano** [comment]
> @gauravano I'd like to open the next check-in after @HarshKhandeparkar's if no one's already booked that spot. Thanks!

Sure @rexagod!

Hey @skilfullycurled, I have updated the above issue body and added the text after the check-in theme for now. Also, fixed the missing link. Thanks!

**Comment by sidntrivedi012** [comment]
Hey everyone, sorry for being late for the check-in. I have a lot of gratitude to express to everyone who has helped me out here. 
Especially, @jywarren and @gauravano who helped me out and guided me whenever I needed to find a way or was stuck in any problem. Thanks a lot for solving all my queries and doubts and for showing me the path whenever I needed guidance. I would also like to thank the whole community at large for collaborating and making this project so awesome. Thank you. 
Cheers to the Public Lab community. Cheers to a better environment. :tada: :100: :smiley: :balloon:

**Comment by HarshKhandeparkar** [comment]
@gauravano where should I open the next check-in?

**Comment by gauravano** [comment]
At https://github.com/publiclab/spectral-workbench

**Comment by HarshKhandeparkar** [comment]
Ok thanks. I will open in an hour or so.

**Comment by skilfullycurled** [comment]
@gauravano, I love that you actually found an accurate context for using for the composition book emoji. 👏

**Comment by ViditChitkara** [comment]
Hello everyone, apologies for being too late!

It has been a great journey with Public Lab since the past one and a half year and there are a lot of people to whom I would like to express my gratitude. @jywarren , @icarito , @ebarry who have always been helpful and polite. @steviepubliclab , @gauravano , @namangupta01 , @jainaman224 , @sagarpreet-chadha who have constantly inspired me and helped me whenever I got stuck anywhere. 

And finally a big thanks to all the community members who have worked really hard making Public lab a great platform!

Thanks!

**Comment by HarshKhandeparkar** [comment]
New check-in open at https://github.com/publiclab/spectral-workbench/issues/451
